lib/rtp_h264/payloader.ex

```diff
@@ -102,8 +96,7 @@ defmodule Membrane.RTP.H264.Payloader do
     type = get_unit_type(payload, state)
     {acc_buffers, state} = handle_accumulator(type, buffer, state)
     {new_buffers, state} = handle_unit_type(type, buffer, state)
-    buffers = timestampify(acc_buffers ++ new_buffers, state)
-    {{:ok, buffer: {:output, buffers}, redemand: :output}, state}
+    {{:ok, buffer: {:output, acc_buffers ++ new_buffers}, redemand: :output}, state}
```

Comment:
What will be the timestamp of outgoing packet if it doesn't have rtp timestamp on entry (e.g. comes from encoder)?

Reply:
it's a job of the serializer now